src/js/entities-service/entities/program-actions.js

```diff
@@ -129,15 +129,25 @@ const Collection = BaseCollection.extend({
       data: JSON.stringify({ data }),
     });
   },
+  isVisibleToCurrentUser(action) {
```

Comment:
Open to any better naming conventions for these functions/variables.

Reply:
I think we can move this function up into the action/flow such that it ends up `const isVisible = action.isVisibleToCurrentUser();`

Reply:
Updated 👍